satpy/tests/test_crefl_utils.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+
+class TestCreflUtils(unittest.TestCase):
+    """Test crefl_utils."""
+
+    def test_get_atm_variables_abi(self):
+        import numpy as np
+        from satpy.composites.crefl_utils import get_atm_variables_abi
+        sphalb, rhoray, TtotraytH2O, tOG = get_atm_variables_abi(0.17690244, 6.123234e-17, 530.61332168, 405.,
+                                                                 21.71342113, 77.14385758, 56.214566960,
+                                                                 0.0043149700000000004, 0.0037296,
+                                                                 0.014107995000000002, 0.052349)
+        if abs(np.array(sphalb) - 0.045213532544630494) >= 1e-10:
+            raise AssertionError('{} is not within {} of {}'.format(np.array(sphalb), 1e-10, 0.045213532544630494))
```

Comment:
Sorry, I should have noticed these earlier. These should be replaced with assert statements or `self.assertX` calls.